test/framework/dump.go

```diff
@@ -190,6 +190,17 @@ func (f *GardenerFramework) dumpGardenerExtensionsInNamespace(ctx context.Contex
 		f.Logger.Println()
 	}
 
+	f.Logger.Infof("%s [EXTENSIONS] [BASTION]", ctxIdentifier)
+	bastions := &v1alpha1.BastionList{}
+	err = k8sClient.Client().List(ctx, bastions, client.InNamespace(namespace))
+	result = multierror.Append(result, err)
+	if err == nil {
+		for _, entry := range bastions.Items {
+			f.dumpGardenerExtension(&entry)
+		}
+		f.Logger.Println()
+	}
+
```

Comment:
This is unrelated to this PR, but kind of snuck into it. Should I remove it?

Reply:
Hm, not sure, @schrodit can you advise, please?

Reply:
Can you give a little more context why do you want to remove it?

In general, the idea of that dump is that in case of an error in an integration test, all available information in the system is printed to the logs.
So that afterwards the failed test can be debugged.

In this case, I guess that the bastion list is mostly empty and will not show up in the logs.
I would rather keep it so that in case there is a bastion resource we are be able to debug it (or at least have the information that there was a bastion resource)

Reply:
OK, thanks @schrodit, then let's keep it.